front/app/components/admin/TabbedResource/index.tsx

```diff
@@ -2,30 +2,19 @@ import React from 'react';
 
 // style
 import styled from 'styled-components';
-import { colors, isRtl } from 'utils/styleUtils';
-
+import { colors, isRtl, fontSizes } from 'utils/styleUtils';
+import { Box, Text } from '@citizenlab/cl2-component-library';
 // typings
 import { ITab } from 'typings';
 
 // components
 import FeatureFlag from 'components/FeatureFlag';
-import { SectionDescription } from 'components/admin/Section';
-import Title from 'components/admin/PageTitle';
 import Tab from './Tab';
 
-const ResourceHeader = styled.div`
-  display: flex;
-  align-items: flex-end;
-  justify-content: space-between;
-  margin-bottom: 30px;
-
-  @media print {
-    margin-bottom: 10px;
-  }
-
-  p {
-    margin-right: 40px;
-  }
+const Title = styled.h1`
```

Comment:
I'm not sure how to do headers using our `Text` component (it doesn't have `h1` as a variant) so I copied the relevant styles from the other file and removed the margin

Reply:
Iva said recommended to use the Title component from the component library yesterday

Reply:
Didn't see that in the docs or on Storybook but it is there 👍